src/builders.ts

```diff
@@ -270,9 +270,21 @@ export class BomBuilder {
     if (!this.reproducible) {
```

Comment:
please turn the bool operator around. 
since we have both cases, the true and false case, we dont need to negate the condition expression in the first place. this is for performance reasons.